schema/bom-1.6.schema.json

```diff
@@ -1635,6 +1635,98 @@
         }
       }
     },
+    "vulnerabilityEvidence": {
+      "type": "object",
+      "title": "Evidence",
+      "description": "Provides the ability to document evidence collected through various forms of extraction or analysis.",
+      "additionalProperties": false,
+      "properties": {
+        "presence": {
+          "type": "array",
+          "description": "Evidence that substantiates the presence or absence of a vulnerability.",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "confidence": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1,
+                "title": "Confidence",
+                "description": "The overall confidence of the evidence from 0 - 1, where 1 is 100% confidence."
+              },
+              "methods": {
+                "type": "array",
+                "title": "Methods",
+                "description": "The methods used to extract and/or analyze the evidence.",
+                "items": {
+                  "type": "object",
+                  "required": [
+                    "technique" ,
+                    "confidence"
+                  ],
+                  "additionalProperties": false,
+                  "properties": {
+                    "technique": {
+                      "title": "Technique",
+                      "description": "The technique used in this method of analysis.",
+                      "type": "string",
+                      "enum": [
+                        "source-code-analysis",
```

Comment:
For most CVEs, the exact technique used by the researcher to identify the CVE would be unknown. In case of vulnerabilities reported in custom code (say using SAST/DAST), then the key data to capture are the rules, policies, queries that helped identify the vulnerability, and some version information for scanners and their engines.